### ISSUE TITLE
Not work on Meteor 1.2.0.1.

### ISSUE THREAD

**Issue by Xiangshen-Meng** [gwendall/meteor-autoform-i18n#7]
I am trying to use meteor-autoform-i18n in my project.
And I tried the  example code, but it does not work without error.

Is it my problem or it's not support 1.2.0.1?

**Comment by sakulstra** [comment]
I also don't get it to work :/

edit: it works when using at least two language files... which is kind of annoying while development ;)

Thanks anyway for this awesome plugin! Continue this great work!

**Comment by Kostanos** [comment]
Confirmed, you should have two languages to make it work.

**Comment by MartinBucko** [comment]
Yes,  translation key must have defined in default 'en' language (i18n/en.i18n.json file) a after this key will be translated to another language. It's tap-i18n package thing and not meteor-autoform-i18n package.

Reference:
https://github.com/TAPevents/tap-i18n#languages-tags-and-translations-prioritization

It take me some time to discover this. Will be fine add this note to README. 
Thanks.

Edit: I have done simple function for export all schemas to JSON string. It can help you prepare your JSON for translating ...
https://gist.github.com/MartinBucko/d9c262d77ca60a67577a 

(it can be merged to package as utility...)
